packages/devtools/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## 0.1.5 - TBD
+* Support expand-all and collapse-all functionality in CPU profiler (Timeline and Performance page).
```

Comment:
nit: I know what this means but it might not be clear to a user. Maybe instead indicate the parts of the ui that are now collapsible.

Reply:
Reworded.